modular_skyrat/modules/bulletrebalance/sniper.dm

```diff
@@ -0,0 +1,9 @@
+/obj/projectile/bullet/p50
+	name =".50 BMG bullet"
+	speed = 0.2
+	damage = 110  //You have 120 health, not a guaranteed instant critical. Unless....
+	paralyze = 0 //Knocks you on your ass hard enough as-is
+	dismemberment = 30
+	armour_penetration = 61 //Bulletproof armor alone will not stop this
+  wound_bonus = 95 //Guaranteed wound, why would it not fuck you up royally?
+	var/breakthings = TRUE
```

Comment:
What is this

Reply:
30% dismemberment chance per shot from 50%
61 armor penetration from 50
95 wound bonus from a whopping 0
var/breakthings unchanged from normal .50 file

Reply:
Ah, if the value remains the same just don't include it here. So you can remove all of the unchanged values.

Reply:
That's all she wrote.